index.html

```diff
@@ -8,181 +8,267 @@
+        </a>
+        <a href="#myCarousel"
+            class="carousel-control-next"
+            role="button"
+            data-slide="next">
+            <span class="sr-only">Next</span>
+            <span class="carousel-control-next-icon"
+                aria-hidden="true"></span>
+        </a>
+        <div id="searchBar"
+            class="m-5">
+            <h1>Bienvenue,</h1>
+            <h5>Des millions de films, émissions télévisées et artistes...</h5>
+            <div class="input-group pt-1">
+                <input type="text"
+                    class="champ rechercher form-control"
+                    placeholder="Taper votre recherche ici ...">
+                <span type="button"
+                    style="background-color:#F2B138"
+                    class="input-group-text search-btn">Rechercher</span>
             </div>
-            <div class="row freeSlider">
-                               <div class="col-4 border border-info">
-                    1er image
-                </div>
-                <div class="col-4 border border-info">
-                    2nd image
-                </div>
-                <div class="col-4 border border-info">
-                    3ème image
-                </div>
-                <div class="col-4 border border-info">
-                    4ème image
+            <div class="message">
+                <p class="zeroText text-danger d-none">* Veuillez saisir un titre svp</p>
+                <p class="notFound text-danger d-none">** Film not found</p>
+            </div>
+
+        </div>
+    </div>
+
+    <section class="container-fluid mt-5 pb-4">
+        <div class="row">
+            <div class="col-12 col-lg-10">
+                <!---------------------- POPULAR MOVIES --------------------->
+                <div class="row">
+                    <div class="content col-12">
+                        <div class="popular">
+                            <h5 class="titles">Populaires</h5>
+                        </div>
+                        <div class="tabs_content">
+                            <div class="addPopularMovies tab_1 rounded active"></div>
+                            <div class="addFreeMovies tab_2"></div>
+                            <div class="addMovies tab_3"></div>
+                            <div class="addSeries tab_4"></div>
+                        </div>
+                    </div>
                 </div>
-                <div class="col-4 border border-info">
-                    5ème image
+                <!---------------------- TRENDING MOVIES --------------------->
+                <div class="row">
+                    <div class="content col-12">
+                        <div class="trending">
+
+                            <h5 class="titles">Tendances</h5>
+                        </div>
+                        <div class="tabs_content">
+                            <div class="addTrendingMovies tab_1 rounded active"></div>
```

Comment:
il faut préférer des classes avec des - 

genre 
```suggestion
                            <div class="add-trending-movies tab-1 rounded active"></div>
```